rainbow-me/rainbow
# Notifications: edit wallet context menu

Fixes TEAM2-80
Figma link (if any):
https://www.figma.com/file/Xc4Pq3CLJNVG52z1K0URvP/notifications?node-id=647%3A4924

## What changed (plus any additional context for devs)

NOTE: see https://github.com/rainbow-me/rainbow/pull/4033 for better commit history– moved to this new branch bc lint mixed in hundreds of extraneous changes 

- converted edit wallet action sheet to context menu
- added deeplink to wallet notification settings
- typescript
- changed pressable area from entire row to just the options icon (approved by Jeremy)

some android issues:
- not sure how to determine appropriate icons or if we even want icons as sf symbols don't work here
- dark mode doesnt work


## Screen recordings / screenshots
https://www.loom.com/share/1c45da21f0f54ca0be2788a179643b08


## What to test
TF 1.7.3 (47)
- make sure all the edit wallet menu options work properly
- make sure the context menu label is identical to the wallet label
- make sure notification settings option only appears when notifications flag is on


## Final checklist
